src/App.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+
+//Create an empty array and store it in a var named todoList
+let todoList = [
+  { "id": 1, 
+  "title": "Workout"
+  },
+  { "id": 2, 
+  "title": "Work"
+  },
+  { "id": 3, 
+  "title": "Work on homework"
+  }
+];
+
+function App() {
+  return (
+    <div>
+      <header>
+      <h2>ToDo List :</h2> 
+        <ul>
+          {todoList.map( function(listItem) {
+            return <li>{listItem.id} {listItem.title}</li>;
```

Comment:
For a `<li>` element, we want to set the unique id to the key attribute.  This will allow us to reference the element later.

`<li key={listItem.id}>{listItem.title}</li>`